File: ios/reclaimapp/TunnelBridge.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(TunnelBridge, NSObject)

RCT_EXTERN_METHOD(startTunnel:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject)

RCT_EXTERN_METHOD(stopTunnel:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject)

RCT_EXTERN_METHOD(getTunnelStatus:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/reclaimapp/TunnelBridge.swift
import Foundation

// TunnelBridge disabled — DNS tunnel approach removed.
// Kept as stub to avoid breaking any existing JS imports.
@objc(TunnelBridge)
class TunnelBridge: NSObject {
  @objc func startTunnel(_ resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) {
    resolve(["success": false, "status": "disabled"])
  }
  @objc func stopTunnel(_ resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) {
    resolve(["success": false, "status": "disabled"])
  }
  @objc func getTunnelStatus(_ resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) {
    resolve(["status": "disabled"])
  }
  @objc static func requiresMainQueueSetup() -> Bool { return false }
}
